projects/canopy/src/styles/variables/_main.scss

```diff
@@ -43,6 +43,9 @@
 
   --keyline-width: 0.1875rem;
 
+  --input-line-height: 1.5;
+  --input-vertical-padding: 0.5625rem; // 9px
```

Comment:
This follows the same approach used for buttons, where we tweak the padding to match the required height. The value `0.5625rem` is the same in both. I did consider creating a new CSS var to contain this value, but I couldn't think of a nice location and name for the var and how it could sit alongside other vars long-term. In the end, I felt that just having the value twice isn't so bad. Happy to chat about this though!

Reply:
you could put these into a new file at `'variables/components/input';`?
I might pass over to @elenagarrone to review and spend a bit more time thinking about it while I am off, but aside from that I'm ok with the approach, and thanks for picking this up @owensgit

Reply:
Thanks @thatguynamedandy, I've created that new file. @elenagarrone also suggested taking a look at the **select** component - which turned out to be the wrong height too, so I've updated that and created `'variables/components/select'`. As the vertical padding values are the same across 3 places now, I've stored that in a new shared variable alongside the other generic form variables.